### PR TITLE
Assignment 1 SE

### PR DESCRIPTION
**Pull request by Essam94**

### CODE REVIEW

**Comment by Essam94** [comment]
Used p3 instead of p6 
Submitted by : Essam Fadel  119523
                          Nadeem Ahmed

File: Image.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    {
+        this.height = height;
+        this.width = width;
+        data = new byte[height * width * 3];
+    }
+    
+    public static void write(String File_Name) throws FileNotFoundException, IOException
+    {
+        FileOutputStream File_Output_Stream_Object = new FileOutputStream(File_Name);
+        File_Output_Stream_Object.write(Image_Hex.getBytes());
```
Comment (floe) [inline Image.java]: This is the location where you should actually write the PPM file, not in the main method (we don't actually need one here).

**Comment by floe** [comment]
The PPM file written by the write() method contains just a few numbers (-1), you would have to move the code from the main method into write (we don't actually need a main method here, because of the test suite). Otherwise, good job!

**Comment by floe** [comment]
P.S. Try to avoid so many static members, they mess up the test suite (as they are shared across all Image objects).

**Pull request closed without merging**